terraform/alerting/slos.tf

```diff
@@ -14,27 +14,36 @@
 
 locals {
   default_per_service_slo = {
-    enable_alert      = false
-    availability_goal = 0.995
-    latency_goal      = 0.99
-    latency_threshold = 60000 # in ms
+    enable_alert           = false
+    availability_goal      = 0.995
+    enable_latency_slo     = false
+    latency_goal           = 0.95
+    latency_threshold      = 60000 # 60 seconds, in ms
+    enable_latency_alert   = false
+    latency_alert_duration = 300000 # 5 minutes, in ms
+
   }
   service_configs = {
     adminapi = merge(local.default_per_service_slo,
-      { enable_alert = true,
+      { enable_alert         = true,
+        enable_latency_alert = true,
+        enable_latency_slo   = false,
```

Comment:
It seems you don't need to set this here, since it is false by default?

Reply:
Yes, thank you.